cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -200,50 +101,275 @@
+       draw lines connecting the branch headers to the line from .org-chart_root.
+    */
+    .respond-to-min( @bp-sm-min, {
+        .grid_column( 6 );
+        position: relative;
+        padding-top: @grid_gutter-width__em;
+        border-width: 0;
+
+        &:before,
+        &:after {
+            content: '';
+            position: absolute;
+            top: 0;
+            width: 50%;
+            height: @grid_gutter-width__em;
+            border: 0 solid @spine-color;
+        }
+
+        &:first-child {
+            padding-right: @grid_gutter-width / 2;
+
+            &:after {
+                left: 50%;
+                border-width: @spine-width__px 0 0 @spine-width__px;
+            }
+        }
+
+        &.last-child {
+            padding-left: @grid_gutter-width / 2;
+
+            &:before{
+                right: 50%;
+                border-width: @spine-width__px @spine-width__px 0 0;
+            }
+
+            .org-chart_node:nth-last-child(2):after {
+                border:none
+            }
+        }
+
+    } );
+
+    &_name {
+        position: relative;
+
+        .respond-to-min(@bp-sm-min, {
+            margin: @grid_gutter-width__em / 2 0;
+            text-align: center;
+        });
+
+        .respond-to-max(@bp-xs-max, {
+            background: @white;
+            margin-bottom: @grid_gutter-width__em;
+
+            &:after {
+                content: '';
+                position: absolute;
+                left: 50%;
+                width: 0;
+                height: @grid_gutter-width__em;
+                border-left: @spine-width__px solid @spine-color;
+            }
+        });
+    }
+}
+
+/* Lines of the org chart are drawn with before & after
+   pseudo elements applied to .org-chart_branches & .org-chart_branch.
+*/
+.org-chart_branches {
+    padding-top: @grid_gutter-width;
+
+    /* :before pseudo element's left border draws a centered vertical line
+       down from .org-chart_root to .org-chart_branches.
+    */
+    .respond-to-min( @bp-sm-min, {
+        position: relative;
+
+        &:before{
+            content: '';
+            position: absolute;
+            top: 0;
+            left: 50%;
+            width: 0;
+            height: @grid_gutter-width__em;
+            border-left: @spine-width__px solid @spine-color;
+        }
+    } );
+}
+
+.org-chart_categories {
+    .link_item {
+        margin-bottom: 0;
+        background: #F8F8F8;
+        border-top: 1px solid @node-border-color;
+        
+        &:last-child {
+            border-bottom: 1px solid @node-border-color;
+        }
+    }
+
+    .link_text {
+       border: none;
+    }
+}
+
+.org-chart_node {
+    .webfont-regular();
+    position:relative;
+    box-sizing: border-box;
+    padding: @grid_gutter-width__em @grid_gutter-width__em / 2;
+    margin: 0 auto @grid_gutter-width__em / 2;
+    border-bottom: @node-border-width__px solid #D1D2D4;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/45cfb87e8ce3b2e74cff956dd73560460cda41c8